doc/developer-guides/sw_design_guidelines.rst

```diff
@@ -73,6 +75,12 @@ below:
 
   - Input data which is only specified by external safety application.
 
+.. note:: If input data can be specified by both of non-safety VM and safety VM,
+   application constraint isn't applicable to these data, related error checking
```

Comment:
```suggestion
   the application constraint isn't applicable to these data.  Related error checking
```

Reply:
Thanks for your comment and review, will update it.